gradio/__init__.py

```diff
@@ -16,8 +16,8 @@
     Carousel,
     Chatbot,
     Checkbox,
-    Checkboxgroup,
     CheckboxGroup,
+    Checkboxgroup,
```

Comment:
Let's get rid of these duplicate imports?

Reply:
I think `Checkboxgroup` is a legacy compatibility alias for `CheckboxGroup`, so getting rid of it would probably break some code out there.

Reply:
You're right!